core/unit_test/TestMathematicalFunctions.hpp

```diff
@@ -240,11 +240,25 @@ struct FloatingPointComparison {
   }
 #if defined(KOKKOS_HALF_T_IS_FLOAT) && !KOKKOS_HALF_T_IS_FLOAT
   KOKKOS_FUNCTION
-  KE::half_t eps(KE::half_t) const { return KE::epsilon<KE::half_t>::value; }
+  KE::half_t eps(KE::half_t) const {
+// FIXME_NVHPC compile-time error
+#ifdef KOKKOS_COMPILER_NVHPC
+    return 0.0009765625F;
+#else
+    return KE::epsilon<KE::half_t>::value;
+#endif
+  }
```

Comment:
`NVPC` really doesn't like the changes in this pull request. Without providing a `float` value here all `MathematicalFunctions` unit tests wouldn't compile for `NVHPC`.